packages/commerce-sdk-react/src/hooks/ShopperBaskets/cache.ts

```diff
@@ -51,9 +51,11 @@ const updateCustomerBasketsQuery = (
     return {
         queryKey: getCustomerBaskets.queryKey({...parameters, customerId}),
         updater: (oldData: CustomerBasketsResult | undefined) => {
-            // do not update if response basket is not part of existing customer baskets
-            if (!oldData?.baskets?.some((basket) => basket.basketId === parameters.basketId)) {
-                return undefined
+            if (!oldData?.baskets?.length) {
```

Comment:
🤔 Actually there are more changes to commerce-sdk-react than I initially thought.

- [ ] determine whether it's going to be a patch or minor release for commerce-sdk-react